xbmc/dbwrappers/dataset.h

```diff
@@ -205,6 +205,7 @@ class Dataset
   Database* db; // info about db connection
   dsStates ds_state; // current state
   Fields *fields_object, *edit_object;
+  std::unordered_map<std::string, unsigned int> name2indexMap;
```

Comment:
I like to annotate the key and value for better clarity, for example something like:

```c++
std::unordered_map<std::string, unsigned int> name2indexMap; // Lower case field name -> database index
```

Reply:
Thanks, I will add the comment.